src/app/tests/suites/TestIcdManagementCluster.yaml

```diff
@@ -208,6 +210,8 @@ tests:
                 value: 1001
               - name: "Key"
                 value: "\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f\xff"
+              - name: "ClientType"
+                value: 0
```

Comment:
Please use symbolic enum names in the YAML.  Much more readable.  So `value: ClientType.Permanent` or `value: ClientType.Ephemeral`.

Reply:
done